### PR TITLE
Add ICMP back to supported AzFw Network protocols

### PR DESCRIPTION
**Pull request by fanymanea**
## Description

ICMP was wrongfully removed from PowerShell in previous release. It was never removed from NRP and, with this change, we are adding it back.

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/packages.config [hunk 1]
```diff
@@ -8,8 +8,8 @@
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.13" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.15" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.14" targetFramework="net452" />
```
Comment (praries880) [inline src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/packages.config]: is this version bump intentional or accidental?

if the change are needed... I see this change taking place in a lot of places as part of this PR that shouldn't see it. Kindly explain why...

**Comment by fanymanea** [comment]
Thank you, guys! I will wait for that merge.

**Comment by praries880** [comment]
@fanymanea  PR https://github.com/Azure/azure-powershell/pull/7575 is merged  ow... you should be able to checkin just the specific changes that you need and not have to make all the other versioning changes. Kindly update the PR.

**Comment by fanymanea** [comment]
Thank you, @cormacpayne !

**Comment by fanymanea** [comment]
@praries880 @cormacpayne 
Guys, can one of you look into this? I thought it was already merged.

**Comment by praries880** [comment]
@fanymanea Kindly look at my last comment... Waiting on you to add a change description to the changelog...

File: src/ResourceManager/Network/Commands.Network/ChangeLog.md [hunk 2]
```diff
@@ -22,12 +22,12 @@
 ## Version 6.9.1
 * Update cmdlet Test-AzureRmNetworkWatcherConnectivity, pass the protocol value to backend.
 * Added ResourceName argument completer to all cmdlets.
-
 * Added WhatIf support to Set-AzureRmNetworkSecurityGroup
 * Added ArgumentCompeter for Subnet's parameter ServiceEndpoints
 * Added cmdlet Get-AzureRmFirewallFqdnTag
 * Added Exclusion list and Global config functionality for application gateway WAF, new cmdlets added
 * Fixed issue with RouteFilter not being set if it was passed to ExpressRouteCircuit Peering as resource
+* Added ICMP back to supported AzureFirewall Network Protocols
```
[the remark this answers is not in the document]
Comment (fanymanea) [reply src/ResourceManager/Network/Commands.Network/ChangeLog.md]: Oh, bad merge with october-release...

**Pull request merged**